README.md

```diff
@@ -61,6 +61,22 @@ We'll be adding more documentation and guides soon! For now you can look to `ref
 
 # Working with the Source
 
+## Testing presubmits locally
+
+To check that you can pass presubmits and nightlies before pushing to PR you can do the following:
+
+- Prerequisites:
+  - Make sure your system can run Android AVDs
+  - Have the build dependencies below installed
+  - Check out the current supported branch of the AAOS Unbundled repo (see the "Check out the Unbundled AAOS Repo" job in `.github/workflows/main` for the correct branch) and set $ORG_GRADLE_PROJECT_unbundledAAOSDir to the path of the checkout
```

Comment:
I'd also mention that you need FIGMA_ACCESS_TOKEN set.

Reply:
Pretty sure that was an accidental change.